server/models/user.js

```diff
@@ -24,6 +24,10 @@ const User = db.define("User", {
       return () => this.getDataValue("password");
     },
   },
+  emailOptIn: {
+    type: DataTypes.BOOLEAN,
+    defaultValue: false,
+  },
 });
```

Comment:
The email scraper is set to just send emails to everyone that has a subscriber email, so right now this boolean isn't really indicating anything on that file. Can you add a check to the emailscraper file to only send to users that have emailOptIn to be true? Let me know if you need help reading my code! I can also do it too if anything.

Reply:
Sure John I can do that. I just wasn't sure if you wanted to be the one to do it so I left it out for now

Reply:
Looks good to me, yeah I didn't mind either way, TY!